Let Explorer Service provide date/time for next test run

laszlourszuly opened the pull request:
Fixes: #11

Commit:
Add "next test date" configuration

Commit:
Expose "next test date" through metrics API

This is a temporary solution in order to quickly get the feature out to
a production environment. Optimally this and a few other parameters
should have their own "/api/status" endpoint.

Commit:
Update UI with next test date/time

blenessy commented on explorer/app2/index.html:
```diff
@@ -59,7 +59,7 @@
                 <div class="row">
                     <div class="col">
                         <div class="counter-header1" style="color: #32354E">Next test run will start at</div>
-                        <div id="next-date-hardcoded" class="counter-header2" style="color: #32354E">13 June, 13:30 (BST) Time in London, UK</div>
```
Considering that the next date on line 186 is different, you are using the same class attribute (next-datetime), which makes me assume that the rendered text will be the **same**. Is this intentional ?

laszlourszuly replied:
Glad that you noticed!

How should we handle "page 0" (`"state": "UNKNOWN"` in the metrics JSON)? Should it be the owner of the `"testNext"` meta attribute in the metrics JSON? Should we then introduce a `"testNextNext"` attribute for the summary page? Right now `"testNext"` means the test that comes after the one that's about to be executed or is currently executing.

laszlourszuly replied:
This means, of course, that "page 0" will show the wrong date/time as is now.

blenessy reviewed (approved):
LGTM, just have a question.

Commit:
Update Bitcoin search input field labels

Commit:
Update test summary description

Adding "Bitcoin was our first crypto love..." text.

Commit:
Change to past tense in post test pages

Commit:
Fix broken grammar

Commit:
Remove hard-coded dates for next test flip boards

Commit:
Clean up code

Commit:
Ensure finish count-down shows correct time

Previously the count-down was reset every time the page was reloaded.
With the current implementation it's ensured it always counts down from
exactly one hour after the test start timestamp.

Commit:
Force ticker to re-sync with test timestamps

Commit:
Merge pull request #26 from radixdlt/fix/form-hint

Fix low-hanging-fruit issues
